src/device/device-management.ts

```diff
@@ -1,66 +1,60 @@
 import * as media from '../media';
+import { LocalAudioTrack, MicrophoneConstraints } from '../media/local-audio-track';
 import { LocalCameraTrack } from '../media/local-camera-track';
 import { LocalDisplayTrack } from '../media/local-display-track';
 import { LocalMicrophoneTrack } from '../media/local-microphone-track';
+import { CameraConstraints, DisplayConstraints } from '../media/local-video-track';
 
-export enum ErrorTypes {
-  DEVICE_PERMISSION_DENIED = 'DEVICE_PERMISSION_DENIED',
-  CREATE_CAMERA_TRACK_FAILED = 'CREATE_CAMERA_TRACK_FAILED',
-  CREATE_MICROPHONE_TRACK_FAILED = 'CREATE_MICROPHONE_TRACK_FAILED',
-}
+import { ErrorTypes, WebrtcError } from '../error/WebrtcError';
 
 /**
- * Represents a WCME error, which contains error type and error message.
+ * Creates MicrophoneTrack and CameraTrack at the same time.
+ *
+ * @param audioConstraints - Audio constraints to create microphone track.
+ * @param videoConstraints - Video constraints to create camera track.
+ * @returns MicrophoneTrack and cameraTrack at same time.
```

Comment:
this should say LocalMicrophoneTrack and LocalCameraTrack (same for comments in line 11)

Reply:
done